pandas/_libs/hashtable_func_helper.pxi.in

```diff
@@ -283,6 +299,8 @@ ctypedef fused htfunc_t:
 
 cpdef value_count(ndarray[htfunc_t] values, bint dropna, const uint8_t[:] mask=None):
     if htfunc_t is object:
+        if lib.is_string_array(values, skipna=True):
```

Comment:
is there a viable way to avoid needing `lib` here?  this file is relatively low-dependency and lib is the opposite